index.js

```diff
@@ -1,10 +1,5 @@
 module.exports = {
-  extends: [
-    'airbnb',
-    'airbnb/hooks',
-    'plugin:@typescript-eslint/recommended',
-    'plugin:prettier/recommended',
-  ],
+  extends: ['airbnb', 'airbnb/hooks', 'plugin:@typescript-eslint/recommended', 'prettier', 'prettier/prettier'],
```

Comment:
Why is prettier still mentioned?

Reply:
I think this this `eslint-config-prettier` which disables all eslint rules related to formatting.